Source/Contrib/TrackViewer/Drawing/DrawArea.cs

```diff
@@ -188,10 +188,10 @@ public void Update()
         public void ZoomReset(DrawTrackDB drawTrackDB)
         {
             if (drawTrackDB == null) return;
-            float minX = drawTrackDB.MinTileX * 2048 - 1024f;
-            float minZ = drawTrackDB.MinTileZ * 2048 - 1024f;
-            float maxX = drawTrackDB.MaxTileX * 2048 + 1024f;
-            float maxZ = drawTrackDB.MaxTileZ * 2048 + 1024f;
+            double minX = drawTrackDB.MinTileX * WorldLocation.TileSize - 1024f;
+            double minZ = drawTrackDB.MinTileZ * WorldLocation.TileSize - 1024f;
+            double maxX = drawTrackDB.MaxTileX * WorldLocation.TileSize + 1024f;
+            double maxZ = drawTrackDB.MaxTileZ * WorldLocation.TileSize + 1024f;
```

Comment:
Why are those calculations switched from float to double?
I doesn't seem related to the switch to 64bit.

Reply:
Good question.

In both the Activity Editor and in RunActivities MapWindow, paths drawn at an angle show distinct wiggles in 64-bit. PerpetualKid (Steffen) found that this change resolved the issue, but it's not clear why.